decidim-conferences/spec/commands/create_conference_spec.rb

```diff
@@ -44,6 +44,7 @@ module Decidim::Conferences
         description: { en: "description" },
         short_description: { en: "short_description" },
         current_organization: organization,
+        organization:,
```

Comment:
Yes, we needed, as here we are using an instance_double ... 

```
    let(:form) do
      instance_double(
        Admin::ConferenceForm,
```
not the pattern that we are used to it: 

```
 let(:form_params) { { .... } }
 let(:form) do
      form_klass.from_params(
        form_params
      ).with_context(
        current_user:,
        current_organization: conference.organization,
        current_participatory_space: conference
      )
    end
```

This would be a refactor outside the scope of this PR.